docs/reference/contrib/modeladmin.rst

```diff
@@ -0,0 +1,227 @@
+
+.. _modeladmin_intro:
+
+=====================
+``ModelAdmin``
+=====================
+
+The ``wagtailmodeladmin`` module allows you to create customisable listing
+pages for any model in your Wagtail project, and add navigation elements to the
+Wagtail admin are so that you can reach them. Simply extend the `ModelAdmin`
```

Comment:
"Wagtail admin **are** so that you can reach them". Presumably mean't to be "area".